src/python/pybind11/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+project(pyxrt)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.0.0)
+
+find_package(pybind11 REQUIRED)
+
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_VERBOSE_MAKEFILE ON)
+set(XILINX_XRT $ENV{XILINX_XRT})
+set(XRT_CORE_LIBRARY xrt_core)
+set(MODE hw)
+
+if (DEFINED ENV{XCL_EMULATION_MODE})
+  set(MODE $ENV{XCL_EMULATION_MODE})
+  string(REPLACE "_" "" XCL_EMU_SUFFIX ${MODE})
+  set(XRT_CORE_LIBRARY xrt_${XCL_EMU_SUFFIX})
+  message("=============== XRT_CORE_LIB=${XRT_CORE_LIBRARY}")
+endif()
+
+find_library(xrt_core_LIBRARY
+  NAMES ${XRT_CORE_LIBRARY}
+  HINTS "${XILINX_XRT}/lib")
+message("xrt_core_LIBRARY=${xrt_core_LIBRARY}")
+
+find_library(xrt_coreutil_LIBRARY
+  NAMES xrt_coreutil
+  HINTS "${XILINX_XRT}/lib")
+message("xrt_coreutil_LIBRARY=${xrt_coreutil_LIBRARY}")
```

Comment:
Linking with XRT core library  is no longer needed.   The coreutil library loads the proper shim library at run-time.